feat(update): Update getting started (#287)
neoforged/Documentation: docs/rendering/feature.md

---
sidebar_position: 1
---
# Features

> A 'feature' in rendering defines a set of objects not baked into the level geometry such as entities, text, and particles. These objects typically have a dynamic position, so things like falling or held block and items also fall into this category. The purpose of the feature renderer is to better batch and order the objects being rendered to the screen. The feature renderer is broken into two phases: the submission phase, where all features are collected; and the rendering phases, where the collected features are rendered.

Reviewer: ```suggestion
A rendering feature defines a set of objects not baked into the level geometry, such as entities, text, and particles. These objects typically have a dynamic position, so things like falling or held blocks and items also fall into this category. The purpose of the feature renderer is then to better batch and order the objects being rendered to the screen. The feature renderer is broken into two phases: the submission phase, where all features are collected; and the rendering phases, where the collected features are rendered.
```
